app/models/agency.py

```diff
@@ -15,7 +23,8 @@ class Agency(db.Model):
     # public user sees a report on the map, the report's agency will only be
     # shown if is_public is True.
     is_public = db.Column(db.Boolean, default=False)
-    users = db.relationship('User', backref='agency', lazy='select')
+    users = db.relationship('User', secondary=agency_user_table,
```

Comment:
could you add a comment that explains that a user can have multiple agencies and that that's why we have this relation?